### PR TITLE
Validates mon_max_pg_per_osd on ceph cluster

### PR DESCRIPTION
**Pull request by parikshithb**
Automates: https://bugzilla.redhat.com/show_bug.cgi?id=1908414
Signed-off-by: Parikshith Byregowda <pbyregow@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/cluster.py [hunk 1]
```diff
@@ -1526,6 +1526,26 @@ def wait_for_silence_ceph_osd_crash_warning(osd_pod_name, timeout=900):
...
+    Args:
+        key (str): Configuration key
+
+    Returns:
+        Configuration value
+
+    """
+    ct_pod = pod.get_ceph_tools_pod()
+    mon_dump_dict = ct_pod.exec_ceph_cmd("ceph mon dump")
+    ceph_mon_name = mon_dump_dict["mons"][0]["name"]
```
Comment (AviadP) [inline ocs_ci/ocs/cluster.py:1542]: as far as i remember, dump might be empty, it better use `mon_dump_dict.get()`

File: ocs_ci/ocs/cluster.py [hunk 2]
```diff
@@ -1526,6 +1526,26 @@ def wait_for_silence_ceph_osd_crash_warning(osd_pod_name, timeout=900):
...
 
+def get_mon_config_value(key):
+    """
+    Gets the default value of a specific ceph monitor config
+
+    Args:
+        key (str): Configuration key
+
+    Returns:
+        Configuration value
```
[the remark this answers is not in the document]
Comment (parikshithb) [reply ocs_ci/ocs/cluster.py]: (any) depends on the key. Added it in the doc string.

File: ocs_ci/ocs/cluster.py (hunk 1 above)
Comment (parikshithb) [reply ocs_ci/ocs/cluster.py]: ack

**Review by PrasadDesala — APPROVED** [review]

**Review by Akarsha-rai — APPROVED** [review]
LGTM

**Review by prsurve — APPROVED** [review]

**Pull request merged**